package-lock.json

```diff
@@ -10,9 +10,9 @@
       "integrity": "sha512-BJ19rjnFFCeopEhbyK2Chg3Tq+o5xkjd6dtKxmFhfjwLH1Il2G7Ha4Jel2hpbyZL2Fh9/vrM9U0bpkANAL3pjA==",
       "dev": true,
       "requires": {
-        "@fimbul/ymir": "^0.6.0",
```

Comment:
I'm confused as to why there are changes to package-lock.json, particularly in the absence of changes to package.json. Are these necessary for the docker setup?

Reply:
I had to update `grpc` package due to a protocol buffers loading bug. accidentally it entirely updated the project.

Reply:
Yeah I think just revert these changes for now then as we discussed. We'll probably want to be updating dependencies, but no reason to mix it up with this PR.